### PR TITLE
fix(har): off-by-one in WebSocket frame header size calculation

### LINKED ISSUE #41142: fix(har): off-by-one in WebSocket frame header size calculation
**Describe the bug**

In `harTracer.ts`, the WebSocket frame header size calculation uses `length > 2 ** 16` to determine when to use the 64-bit extended payload length field. Per [RFC 6455 Section 5.2](https://www.rfc-editor.org/rfc/rfc6455#section-5.2), a 16-bit unsigned integer can represent values 0–65535, so a payload of exactly 65536 (`2 ** 16`) bytes cannot fit in 16 bits and requires the 64-bit extended length field (8 additional bytes). The current condition misses this boundary, causing `response._transferSize` to be 6 bytes too small for payloads of exactly 65536 bytes.

**Steps to reproduce**

Record HAR for a WebSocket connection that receives a frame with exactly 65536 bytes of payload data. The reported `response._transferSize` will be 6 bytes less than the actual on-wire size.

**Expected behavior**

The condition should be `length >= 2 ** 16` instead of `length > 2 ** 16`.

**Introduced in**

Commit 426cc4e9257b ("feat(har): calculate `request.headerSize`, `response.headerSize`, and `response._transferSize` for `WebSocket`")

### PR DESCRIPTION
**Pull request by yen0304**
## Summary
- Fix boundary condition in WebSocket frame header size calculation: `length > 2 ** 16` → `length >= 2 ** 16`
- Per RFC 6455 Section 5.2, a payload of exactly 65536 bytes requires the 64-bit extended length field (16-bit unsigned max is 65535)
- Add test for the exact boundary (2**16 bytes payload)

Fixes https://github.com/microsoft/playwright/issues/41142

### CODE REVIEW

File: tests/library/har-websocket.spec.ts [hunk 1]
```diff
@@ -232,6 +232,34 @@ it('should include gigantic websocket messages', async ({ contextFactory, server
   expect(messages[0].time).toBeLessThanOrEqual(messages[1].time);
 });
 
+it('should use 64-bit extended length for exactly 2**16 byte websocket payload', async ({ contextFactory, server }, testInfo) => {
+  // RFC 6455 Section 5.2: 16-bit extended length covers 126-65535.
+  // Payloads of exactly 65536 (2**16) bytes require the 64-bit extended length field.
+  const incoming = 'x'.repeat(2 ** 16);
```
Comment (dcrousso) [inline tests/library/har-websocket.spec.ts:238]: rather than adding an entirely new test for this, lets just update the test right above
```diff
-const incoming = 'x'.repeat(2 ** 16 + 1);
+const incoming = 'x'.repeat(2 ** 16);
```

**Review by dcrousso — APPROVED** [review]
nice catch!

**Review by yury-s — APPROVED** [review]
